tendrl/node_agent/flows/flow_execution_exception.py

```diff
@@ -0,0 +1,4 @@
+class FlowExecutionFailedError(Exception):
```

Comment:
I have retained this as a place holder exception that I expect flows to raise if something goes wrong. This is handled in RPC layer.